src/makeCalculator.js

```diff
@@ -37,7 +37,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(arg) {
+      this.result += arg;
+    },
+    operate(callback, arg) {
+      callback.call(this, arg);
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    subtract(arg) {
+      this.result -= arg;
+    },
+    multiply(arg) {
+      this.result *= arg;
+    },
```

Comment:
```suggestion
    },
    
```
Consider adding an empty line between methods to improve readability